spectec/src/main/watsup.ml

```diff
@@ -1,3 +1,5 @@
+open Spectec
```

Comment:
Is this needed?

Reply:
Yes because I split the code between a library and an executable. I believe this will ease things in the future when we don't have a single binary and people start building many different tools around the library part.

I could add a `(wrapped false)` on the `spectec` library so that it's not wrapped in a `Spectec` module but then, linking with that library would directly expose modules such as `Ast`, `Lexer`, `Lib` or `Parse(r)` which are quite common and will cause some conflict... Whereas now, you have to write `Spectest.Ast` to access them.